modules/aerodyn/src/FVW_Subs.f90

```diff
@@ -481,11 +480,11 @@ logical function have_nan(p, m, x, z, u, label)
          have_nan=.True.
       endif
       if (any(isnan(x%W(iW)%Eps_NW))) then
-         print*,trim(label),'NaN in E_NW'
+         print*,trim(label),'NaN in E_NW'//trim(num2lstr(iW))
```

Comment:
Could we change the `print*` to `call WrScr` instead?  The `print*` doesn't show up properly with Simulink.

Reply:
Those are debug print statements, and are never called, so I'm not worrying about them

Reply:
Perfect!  No need to worry about them then.  [apologies for my laziness in not checking the context better]